infrastructure/repository/labels-workflow.tf

```diff
@@ -1,15 +1,50 @@
 variable "workflow_labels" {
```

Comment:
my only concern is that some of the more urgent labels (bug/regression/crash) seem to blend in quite easily with the less urgent (documentation/needs-triage) so glancing over the issue list requires more 👀 ...or could just be my screen brightness 😅 i do feel though that "proposal" should get its own color as its more rare/lives in a different category imo to "documentation"